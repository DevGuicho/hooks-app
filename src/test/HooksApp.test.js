import { shallow } from "enzyme";
import HooksApp from "../HooksApp";

describe("Pruebas del componente <HookApp />", () => {
  test("debe mostrar correctamenete el componente", () => {
    const wrapper = shallow(<HooksApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
