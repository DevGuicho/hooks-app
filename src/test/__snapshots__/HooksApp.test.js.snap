// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas del componente <HookApp /> debe mostrar correctamenete el componente 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
